src/python/Publisher/TaskPublishGo.py

```diff
@@ -309,7 +314,7 @@ def checkBlockMigration(taskname, migrateApi, block, migLogDir):
         migCreation = datetime.fromtimestamp(creationDate).strftime('%Y-%m-%d,%H:%M:%S')
         # FiledMigFile format is CSV: creationDate, block, taskname
```

Comment:
please update this comment as well

Reply:
I mean Line 310. SOmehow the highlight is in the wrong line

Reply:
Uh, I messed it, thank you!

Reply:
+1